packages/wonder-blocks-dropdown/src/components/__docs__/multi-select.stories.js

```diff
@@ -7,10 +7,195 @@ import {View} from "@khanacademy/wonder-blocks-core";
 import type {Labels} from "@khanacademy/wonder-blocks-dropdown";
 import type {StoryComponentType} from "@storybook/react";
 
-import {MultiSelect, OptionItem} from "../../index.js";
+import Button from "@khanacademy/wonder-blocks-button";
+import Color from "@khanacademy/wonder-blocks-color";
+import {MultiSelect, OptionItem} from "@khanacademy/wonder-blocks-dropdown";
+import {Checkbox} from "@khanacademy/wonder-blocks-form";
+import {OnePaneDialog, ModalLauncher} from "@khanacademy/wonder-blocks-modal";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import {HeadingLarge} from "@khanacademy/wonder-blocks-typography";
+
+import ComponentInfo from "../../../../../.storybook/components/component-info.js";
+import {name, version} from "../../../package.json";
+import multiSelectArgtypes from "./base-select.argtypes.js";
+import {defaultLabels} from "../../util/constants.js";
 
 export default {
     title: "Dropdown / MultiSelect",
+    component: MultiSelect,
+    argTypes: {
+        ...multiSelectArgtypes,
+        labels: {
+            defaultValue: defaultLabels,
+        },
+    },
+    args: {
+        isFilterable: false,
+        opened: false,
+        disabled: false,
+        light: false,
+        shortcuts: false,
+        implicitAllEnabled: false,
+        id: "",
+        testId: "",
+    },
+    decorators: [
+        (Story: StoryComponentType): React.Element<typeof View> => (
+            <View style={styles.example}>
+                <Story />
+            </View>
+        ),
```

Comment:
Does this behave differently that adding the `<View style={styles.example}>` as a wrapper in the `Template` below?